assets/js/customize.js

```diff
@@ -1,19 +1,58 @@
-/* global twentytwentyoneGetHexLum */
+/* global twentytwentyoneGetHexLum, backgroundColorNotice */
 
 ( function() {
 	// Wait until the customizer has finished loading.
 	wp.customize.bind( 'ready', function() {
+		var supportsDarkMode = ( 127 < twentytwentyoneGetHexLum( wp.customize( 'background_color' ).get() ) && wp.customize( 'respect_user_color_preference' ).get() );
```

Comment:
Should this technically be `<=` to cover all possibilities and be consistent with the logic [here](https://github.com/WordPress/twentytwentyone/blob/trunk/functions.php#L583)?

```suggestion
		var supportsDarkMode = ( 127 <= twentytwentyoneGetHexLum( wp.customize( 'background_color' ).get() ) && wp.customize( 'respect_user_color_preference' ).get() );
```

Reply:
Makes sense, committed :+1: